src/server/create/env-test-suite/controllers/create.js

```diff
@@ -13,7 +15,7 @@ const envTestSuiteCreateController = {
         scope: [config.get('oidcAdminGroupId'), '{payload.teamId}']
       }
     },
-    pre: [provideCreate]
+    pre: [provideCreate, provideAuthedUser]
```

Comment:
is this needed? provideAuthedUser

Reply:
Yes, because the audit refers to `{request.pre?.authedUser.id}`

Reply:
see below comment I don't think we need this. Its available elsewhere